examples/hosts/app-integration/external-data/src/utilities/webhook.ts

```diff
@@ -16,74 +15,48 @@ import { ExternalDataSource } from "./externalData";
 export type SubscriberUrl = string;
 
 /**
- * Mock implementation of a webhook configured to receive updates from an external data provider.
+ * Mock webhook manager.
+ * Can be registered with to receive notifications when {@link MockWebhook.notifySubscribers} is called.
+ *
+ * @typeParam TData - The kind of data that will be sent alongside notifications to subscribers.
+ * Must be JSON-serializable.
  */
-export class MockWebhook implements IDisposable {
+export class MockWebhook<TData = unknown> implements IDisposable {
```

Comment:
I don't think I saw any `dispose()` calls for the webhook in the services... this beig an example app, I'd say maybe we don't need to complicate it with `IDisposability`, and if anything be clearer about it not being production-ready code.

Reply:
Yeah, I think I agree. I'll go ahead and clean that up, since I'm already making substantial changes here.